lua/lvim/lsp/null-ls/init.lua

```diff
@@ -9,14 +9,7 @@ function M:setup()
     return
   end
 
-  null_ls.config(lvim.lsp.null_ls.config)
-  local default_opts = require("lvim.lsp").get_common_opts()
-
-  if vim.tbl_isempty(lvim.lsp.null_ls.setup or {}) then
-    lvim.lsp.null_ls.setup = default_opts
-  end
-
-  require("lspconfig")["null-ls"].setup(lvim.lsp.null_ls.setup)
+  null_ls.setup(lvim.lsp.null_ls.config)
```

Comment:
```suggestion
  local default_opts = require("lvim.lsp").get_common_opts()
  null_ls.setup(vim.tbl_deep_extend("force", default_opts, lvim.lsp.null_ls.setup))
```
And then we can remove `lvim.lsp.null_ls.config` which hopefully is seldom used anyway.